tests/test_peers_api.py

```
@@ -0,0 +1,129 @@
+import json
+
+from tests.helpers import MockRequest, MockRow, MockDB, make_env, make_stmt, load_worker
+
+worker = load_worker()
+peers = worker._get_peers_module()
+
+JWT = "test-jwt-secret"
+ENC = "test-encryption-key"
+
+
+def _parse(resp):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_parse`

(ANN202)